### PR TITLE
[Upmeter] Turn off smoke-mini storage class

### PR DESCRIPTION
**Pull request by shvgn**
Signed-off-by: Eugene Shevchenko <evgeny.shevchenko@flant.com>

## Description

Turning off smoke-mini storage class in clusters.

## Why do we need it, and what problem does it solve?

Smoke-mini checks disks mounting across nodes. We don't track its reliability for now. By turning
off storage class, we avoid further issues related to extra Released PVs until we are ready to fully
support the reliability of disk subsystems.

## Changelog entries

```changes
section: upmeter
type: chore
summary: Turned off smoke-mini storage classes, so it will not order PVs by default and only serve us network checks.
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ help:
 	@awk 'BEGIN {FS = ":.*##"; printf "Usage: make ${FORMATTING_BEGIN_YELLOW}<target>${FORMATTING_END}\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  ${FORMATTING_BEGIN_BLUE}%-46s${FORMATTING_END} %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 
-GOLANGCI_VERSION = 1.42.0
```
Comment (shvgn) [inline Makefile:25]: Linter did not auto-fix due to dependencies issues. So I upgraded it. And it made me fix a couple of its complaints. It should be a separate PR, but I decided to try a shortcut.

File: ee/modules/600-flant-integration/hooks/license_test.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ var _ = Describe("Flant integration :: hooks :: license ::", func() {
 			registry := rand.String(8)
 			auth := getConfig()
 			passwordWithNoSpaces := auth.Password
-			auth.Password = auth.Password + "\n"
+			auth.Password += "\n"
```
Comment (shvgn) [inline ee/modules/600-flant-integration/hooks/license_test.go:92]: Linter complaint

File: modules/040-node-manager/hooks/discover_standby_ng_test.go [hunk 3]
```diff
@@ -473,7 +473,7 @@ status:
 		BeforeEach(func() {
 			state := nodeGroupWithoutZones
 			for i := 1; i <= 12; i++ {
-				state = state + fmt.Sprintf(nodeWorkerTemplate, i)
+				state += fmt.Sprintf(nodeWorkerTemplate, i)
```
Comment (shvgn) [inline modules/040-node-manager/hooks/discover_standby_ng_test.go:476]: Linter complaint

File: modules/500-upmeter/openapi/config-values.yaml [hunk 4]
```diff
@@ -158,11 +158,12 @@ properties:
               - [ "1.1.1.1/32" ]
             description: An array if CIDRs that are allowed to authenticate.
       storageClass:
+        default: false
```
Comment (shvgn) [inline modules/500-upmeter/openapi/config-values.yaml:161]: The change of interest. Other things in the schema is by Goland formatter.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/aws'...

**Review by nabokihms — APPROVED** [review]
